web-common/src/features/dashboards/dashboard-utils.ts

```diff
@@ -22,12 +23,44 @@ const countRegex = /count(?=[^(]*\()/i;
 const sumRegex = /sum(?=[^(]*\()/i;
 
 export function isSummableMeasure(measure: MetricsViewSpecMeasure): boolean {
+  if (measure.validPercentOfTotal) return true;
```

Comment:
Do we get `validPercentOfTotal=false` when nothing is set in yaml? If not then maybe we should support explicitly setting `valid_percent_of_total: false` here. That way if there is some complex expression then users can have a fallback.

Reply:
I think the default for `valid_percent_of_total` is false.